.github/workflows/publish_debian.yml

```diff
@@ -47,5 +48,23 @@ jobs:
         with:
           context: base/debian
           platforms: linux/arm64, linux/amd64, linux/386
-          push: ${{ github.event_name != 'pull_request' }}
+          push: ${{ github.event_name == 'push' }}
           tags: ghcr.io/railwayapp/nixpacks:debian, ghcr.io/railwayapp/nixpacks:latest, ghcr.io/railwayapp/nixpacks:debian-${{ steps.date.outputs.date }}
+      -
+        name: Bump base image
+        if: ${{ github.event_name == 'push' }}
+        run: |
+          sed -i 's/nixpacks:debian-.*/nixpacks:debian-${{ steps.date.outputs.date }}";/g' src/nixpacks/images.rs
+      - 
+        name: Create Pull Request
```

Comment:
Do you think we need to make a pull request? This could very well just be a commit.